internal/pkg/mapper/orchestrator/local_load.go

```diff
@@ -197,21 +197,36 @@ func (l *localLoader) parseTaskConfig(task *model.Task) error {
 		errors.Append(err)
 	}
 
-	// Get target config path
-	targetConfigPath, err := parser.configPath()
+	// Get enabled
+	task.Enabled, err = parser.enabled()
 	if err != nil {
 		errors.Append(err)
 	}
 
-	// Get target config
-	targetConfig, err := l.getTargetConfig(targetConfigPath)
+	// Get target config path
+	targetConfigPath, err := parser.configPath()
 	if err != nil {
-		errors.Append(err)
-	} else if targetConfig != nil {
-		task.ComponentId = targetConfig.ComponentId
-		task.ConfigId = targetConfig.Id
-		task.ConfigPath = l.MustGet(targetConfig.Key()).Path()
-		markConfigUsedInOrchestrator(targetConfig, l.config)
+		if task.Enabled {
+			errors.Append(err)
+		}
```

Comment:
Na disk sa neuklada pár `configId` a `componentId` ale `configPath`.
Tj. je to `path` na disku, ku konfiguracii, ktoru ma task spustit.

Ak je `enabled:false`, tak `configPath` moze chybat.